aviary/models/multi_engine_single_aisle/multi_engine_single_aisle_data.py

```diff
@@ -0,0 +1,433 @@
+
+# Fins
+# ---------------------------
+inputs.set_val(Aircraft.Fins.AREA, 0.0, 'ft**2')
+inputs.set_val(Aircraft.Fins.NUM_FINS, 0)
+inputs.set_val(Aircraft.Fins.TAPER_RATIO, 10.0)
+inputs.set_val(Aircraft.Fins.MASS, 1.0, 'lbm')
+inputs.set_val(Aircraft.Fins.MASS_SCALER, 1.0)
+
+# Fuel
+# ---------------------------
+inputs.set_val(Aircraft.Fuel.AUXILIARY_FUEL_CAPACITY, 0.0, 'lbm')
+inputs.set_val(Aircraft.Fuel.DENSITY_RATIO, 1.0)
+inputs.set_val(Aircraft.Fuel.FUEL_SYSTEM_MASS_SCALER, 1.0)
+inputs.set_val(Aircraft.Fuel.NUM_TANKS, 7)
+inputs.set_val(Aircraft.Fuel.TOTAL_CAPACITY, 46063.0, 'lbm')
+inputs.set_val(Aircraft.Fuel.UNUSABLE_FUEL_MASS_SCALER, 1.0)
+
+# Furnishings
+# ---------------------------
+inputs.set_val(Aircraft.Furnishings.MASS_SCALER, 1.0)
+
+# Fuselage
+# ---------------------------
+inputs.set_val(Aircraft.Fuselage.NUM_FUSELAGES, 1)
+inputs.set_val(Aircraft.Fuselage.LENGTH, 124.75, 'ft')
+inputs.set_val(Aircraft.Fuselage.PASSENGER_COMPARTMENT_LENGTH, 98.5, 'ft')
+inputs.set_val(Aircraft.Fuselage.MAX_HEIGHT, 13.0208, 'ft')
+inputs.set_val(Aircraft.Fuselage.MAX_WIDTH, 12.33, 'ft')
+inputs.set_val(Aircraft.Fuselage.MILITARY_CARGO_FLOOR, False)
+inputs.set_val(Aircraft.Fuselage.PLANFORM_AREA, 124.75 * 12.33, 'ft**2')
+inputs.set_val(Aircraft.Fuselage.MASS_SCALER, 1.0)
+inputs.set_val(Aircraft.Fuselage.WETTED_AREA, 4142.317, 'ft**2')  # Override
+inputs.set_val(Aircraft.Fuselage.WETTED_AREA_SCALER, 1.0)
+
+# Horizontal Tail
+# ---------------------------
+inputs.set_val(Aircraft.HorizontalTail.AREA, 407.335370699457, 'ft**2')
+inputs.set_val(Aircraft.HorizontalTail.ASPECT_RATIO, 5.444)
+inputs.set_val(Aircraft.HorizontalTail.TAPER_RATIO, 0.3008)
+inputs.set_val(Aircraft.HorizontalTail.THICKNESS_TO_CHORD, 0.1195)
+inputs.set_val(Aircraft.HorizontalTail.VERTICAL_TAIL_FRACTION, 0.0)
+inputs.set_val(Aircraft.HorizontalTail.MASS_SCALER, 1.0)
+inputs.set_val(Aircraft.HorizontalTail.WETTED_AREA, 707.706, 'ft**2')  # Override
+inputs.set_val(Aircraft.HorizontalTail.WETTED_AREA_SCALER, 1.0)
+
+# Hydraulics
+# ---------------------------
+inputs.set_val(Aircraft.Hydraulics.SYSTEM_PRESSURE, 3000., 'lbf/ft**2')
+inputs.set_val(Aircraft.Hydraulics.MASS_SCALER, 1.0)
+
+# Instruments
+# ---------------------------
+inputs.set_val(Aircraft.Instruments.MASS_SCALER, 1.0)
+
+# Landing Gear
+# ---------------------------
+inputs.set_val(Aircraft.LandingGear.CARRIER_BASED, False)
+inputs.set_val(Aircraft.LandingGear.MAIN_GEAR_OLEO_LENGTH, 84.0, 'inch')
+inputs.set_val(Aircraft.LandingGear.MAIN_GEAR_MASS_SCALER, 1.0)
+inputs.set_val(Aircraft.LandingGear.NOSE_GEAR_OLEO_LENGTH, 58.8, 'inch')
+inputs.set_val(Aircraft.LandingGear.NOSE_GEAR_MASS_SCALER, 1.0)
+
+# Nacelle
+# ---------------------------
+inputs.set_val(Aircraft.Nacelle.AVG_DIAMETER, 7.0, 'ft')
+inputs.set_val(Aircraft.Nacelle.AVG_LENGTH, 11.65, 'ft')
+inputs.set_val(Aircraft.Nacelle.MASS_SCALER, 1.0)
+inputs.set_val(Aircraft.Nacelle.WETTED_AREA_SCALER, 1.0)
```

Comment:
I just used the same values for both engines so strictly speaking they don't have to be vectors (they get vectorized during the preprocessing step). Since this isn't a real aircraft I didn't worry about the details too much

Reply:
even though this isn't a real aircraft (and the numbers don't have to be "good"), it is an example users will reference when building their own models, so I think we should demonstrate how a vehicle with multiple engine types would be set up.